### ISSUE TITLE
Error when rebuilding docker

### ISSUE THREAD

**Issue by paulormart** [ibp-network/ibp-monitor#109]
Rebuilding the monitor from docker got me this error when `docker compose -f docker-compose-alerts.yml up`
```
> [4/9] RUN npm install -g npm:
#0 0.733 npm ERR! code EBADENGINE
#0 0.735 npm ERR! engine Unsupported engine
#0 0.735 npm ERR! engine Not compatible with your version of node/npm: npm@10.0.0
#0 0.735 npm ERR! notsup Not compatible with your version of node/npm: npm@10.0.0
#0 0.735 npm ERR! notsup Required: {"node":"^18.17.0 || >=20.5.0"}
#0 0.735 npm ERR! notsup Actual:   {"npm":"8.19.4","node":"v16.20.2"}
#0 0.736 
#0 0.736 npm ERR! A complete log of this run can be found in:
#0 0.736 npm ERR!     /root/.npm/_logs/2023-09-07T21_35_22_567Z-debug-0.log
------
failed to solve: process "/bin/sh -c npm install -g npm" did not complete successfully: exit code: 1
```

The workaround is to upgrade the node version at `Dockerfile.ibp.services` from `FROM node:16-slim` to `FROM node:20-slim`

@dcolley @kukabi do you have the same issue?

**Comment by dcolley** [comment]
I have not built these images for some time. Do you know what library is causing the error?

**Comment by paulormart** [comment]
fixed by PR #110
